packages/react-app/src/views/BuildsListView.jsx

```diff
@@ -16,17 +46,156 @@ export default function BuildsListView() {
+          description: "Submission Error. You don't have the required role.",
+          variant: toastVariant,
+        });
+        setIsSubmitting(false);
+        return;
+      }
+      toast({
+        status: "error",
+        description: "Submission Error. Please try again.",
+        variant: toastVariant,
+      });
+      setIsSubmitting(false);
+      return;
+    }
+
+    toast({
+      status: "success",
+      description: "Build submitted! It will show up once it's reviewed and accepted",
+      variant: toastVariant,
+    });
+    clearForm();
+    setIsSubmitModalOpen(false);
+    setIsSubmitting(false);
+  };
+
   return (
     <Container maxW="container.xl" centerContent>
       <Heading as="h2">All builds</Heading>
       <Text color={secondaryFontColor} mb="6">
         Explore all our Ethereum web3 projects.
       </Text>
+      <Button colorScheme="blue" mb={8} onClick={() => setIsSubmitModalOpen(true)}>
+        Submit new build
+      </Button>
       <SimpleGrid columns={[1, null, 2, null, 3]} spacing={6} pb={20}>
         {builds.map(build => (
           <BuildCard build={build} key={build.name} />
         ))}
       </SimpleGrid>
+      <Modal isOpen={isSubmitModalOpen} onClose={() => setIsSubmitModalOpen(false)}>
+        <ModalOverlay />
+        <ModalContent>
+          <ModalHeader>New build form</ModalHeader>
+          <ModalCloseButton />
+          <ModalBody px={8} pb={8}>
+            <FormControl mb={4} isRequired isInvalid={errors.buildName}>
+              <FormLabel htmlFor="buildName">Build name</FormLabel>
+              <Input
+                id="buildName"
+                placeholder="Build name"
+                value={buildName}
+                onChange={evt => setBuildName(evt.target.value)}
+              />
+              <FormErrorMessage>This field is required</FormErrorMessage>
+            </FormControl>
+            <FormControl mb={4} isRequired isInvalid={errors.description}>
+              <FormLabel htmlFor="description">Description</FormLabel>
+              <Textarea
+                id="description"
+                placeholder="Write a short description for this build"
+                value={description}
+                onChange={evt => setDescription(evt.target.value)}
+              />
+              <FormErrorMessage>This field is required</FormErrorMessage>
+            </FormControl>
+            <FormControl mb={4} isRequired isInvalid={errors.buildUrl}>
+              <FormLabel htmlFor="buildUrl">Branch url</FormLabel>
+              <Input
+                id="buildUrl"
+                placeholder="https://..."
+                value={buildUrl}
+                onChange={evt => setBuildUrl(evt.target.value)}
+              />
+              <FormErrorMessage>This field is required</FormErrorMessage>
+            </FormControl>
+            <FormControl mb={4} isInvalid={errors.imageUrl}>
+              <FormLabel htmlFor="imageUrl">Build name</FormLabel>
```

Comment:
This should be Image URL

Reply:
Fixed on https://github.com/moonshotcollective/scaffold-directory/pull/109/commits/b0c044228782cc3fb9dcec574d52d55018fec19e